pages/common/git-authors.md

```diff
@@ -0,0 +1,17 @@
+# git authors
+
+> Display log or list of committers of a Git repository. Everything except `--list` will create a new file called `AUTHORS`.
```

Comment:
Perhaps this will read better?
```suggestion
> Display a log or list of committers of a Git repository. Everything except `--list` will create a new file called `AUTHORS`.
```